ibis/pandas/api.py

```diff
@@ -1,9 +1,36 @@
 from __future__ import absolute_import
 
+import ibis
+import toolz
+
 from ibis.pandas.client import PandasClient
 from ibis.pandas.decimal import execute_node  # noqa: F401
 from ibis.pandas.execution import execute  # noqa: F401
 
 
+__api__ = 'connect', 'execute'
+
+
 def connect(dictionary):
     return PandasClient(dictionary)
+
+
+def _flatten_subclass_tree(cls):
+    subclasses = frozenset(cls.__subclasses__())
+    children = frozenset(toolz.concat(map(_flatten_subclass_tree, subclasses)))
+    return frozenset({cls}) | subclasses | children
+
+
+class PandasExprTranslator(object):
+    _registry = frozenset(toolz.concat(
+        _flatten_subclass_tree(types[0]) for types in execute_node.funcs
```

Comment:
This is kind of a nifty side of effect of using `multipledispatch`: we can flatten the type tree of the first argument of every `execute_node` rule and see what operations pandas implements.

Reply:
Cool, I will document this code as well.